go/dbproxy/dbproxy.go

```diff
@@ -43,7 +43,7 @@ func Run(args []string) {
 	r.GET("/localtable/:name/:userid", localTableGetAll)
 	r.GET("/localtable/:name/:userid/:uniqueid", localTableGetOne)
 	r.DELETE("/localtable/:name/:userid/:uniqueid", localTableDeleteOne)
-	r.POST("/localtable/:name/:userid", localTableInsertOne)
+	r.POST("/localtable/:name/:userid/:uniqueid", localTableInsertOne)
```

Comment:
uniuqeids can be long and not URL safe. I think it's better to pass them in the POST body.